<?php

class User{
	
	const TABLE='users';
	const DBNAME='vk';

	public $db = false;
	public $result = false;


	public function _construct(){
		$this->$db = new PDO('mysql:host=localhost;dbname= ' .self::DBNAME';charset=utf8', 'root', '');
		$this->db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXEPTION);
	}

	public function registration($data){
		$insert = $this->$db->prepare('INSERT into users (email, login, password,sex,numberofgames,dob,historical,party,economic,sf,rpg,allgames) values ("' . $_POST['email'] . '", "' . $_POST['login'] . '", "' . $_POST['password'] . '", "' . $_POST['pol'] . '", "' . $_POST['numberofgames'] . '", "' . $_POST['dob'] .'", "' . $_POST['historical']. '" , "' . $_POST['party'] . '", "' . $_POST['economic'] .'", "' . $_POST['sf'] .'", "' . $_POST['rpg'] . '", "' . $_POST['all'] .'")');


		try{
			$insert->execute();
			$this->result= 'Hello' . $data['login'];
		}
		catch(Exeption $e){
			$this->result = $e->getMessage();
		}

	}
}